connectivity/check/context.go

```diff
@@ -553,16 +554,36 @@ func (ct *ConnectivityTest) report() error {
 	}
 
 	if ct.params.Perf {
-		// Report Performance results
-		ct.Headerf("🔥 Performance Test Summary: ")
-		ct.Logf("%s", strings.Repeat("-", 145))
-		ct.Logf("📋 %-15s | %-50s | %-15s | %-15s | %-15s | %-15s", "Scenario", "Pod", "Test", "Num Samples", "Duration", "Avg value")
-		ct.Logf("%s", strings.Repeat("-", 145))
-		for p, d := range ct.PerfResults {
-			ct.Logf("📋 %-15s | %-50s | %-15s | %-15d | %-15s | %.2f (%s)", d.Scenario, p.Pod, p.Test, d.Samples, d.Duration, d.Avg, d.Metric)
-			ct.Debugf("Individual Values from run : %s", d.Values)
+
+		ct.Headerf("🔥 Latency Test Summary: ")
+
+		// Report Performance results for latency
+		if ct.params.PerfLatency {
```

Comment:
nit: don't we only want to print that header in case of latency tests and `ct.Headerf("🔥 Performance Test Summary: ")` in the `else` case below (see other review remark)?

```suggestion
		// Report Performance results for latency
		if ct.params.PerfLatency {
			ct.Headerf("🔥 Latency Test Summary: ")
```